src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -1161,8 +1161,11 @@ public function single_row( $item ) {
 				case 'description':
 					$classes = 'column-description desc';
 
-					echo "<td class='$classes{$extra_classes}'>
-						<div class='plugin-description'>$description</div>
+					echo "<td class='$classes{$extra_classes}'>";
+					if ( WP_Plugin_Dependencies::has_dependencies( $plugin_file ) ) {
+						$this->add_dependencies_warning( $plugin_file );
+					}
+					echo "<div class='plugin-description'>$description</div>
```

Comment:
> `"<td class='$classes{$extra_classes}'>";`

While this is technically and WPCS wise fine, it still looks wrong to me.

In lines 1597 you've used a `printf()` and I think refactoring these echos to `printf()` would improve readability and code quality here.